docs/output.md

```diff
@@ -129,6 +129,18 @@ NanoPlot is used to calculate various metrics and plots about the quality and le
 
 </details>
 
+## Digital normalization with BBnorm
+
+If the pipeline is called with the `--bbnorm` option, it will normalize sequencing depth by removing reads to 1) reduce coverage of very abundant kmers and 2) delete very rare kmers (see `--bbnorm_target` and `--bbnorm_min` parameters).
```

Comment:
```suggestion
If the pipeline is called with the `--bbnorm` option, it will normalize sequencing depth of libraries prior assembly by removing reads to 1) reduce coverage of very abundant kmers and 2) delete very rare kmers (see `--bbnorm_target` and `--bbnorm_min` parameters).
```